SwiftVoiceQuickstart.xcodeproj/project.pbxproj

```diff
@@ -306,10 +327,11 @@
 				CODE_SIGN_ENTITLEMENTS = SwiftVoiceQuickstart/SwiftVoiceQuickstart.entitlements;
 				DEVELOPMENT_TEAM = "";
 				INFOPLIST_FILE = SwiftVoiceQuickstart/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
I didn't see similar change for ObjC QS,  is that intentional ?

Reply:
ObjC compiler isn't that restrict as Swift so the code still compiles. Will make the deploy target the same version.